bookshelf.js

```diff
@@ -0,0 +1,71 @@
+
+
+let bttn = document.createElement( 'button' );
+bttn.setAttribute( 'id', 'fetch-books-btn' );
+
+let text = document.createTextNode( 'Fetch Books' );
+bttn.appendChild( text );
+bttn.addEventListener( 'click', fetchBooks );
+
+document.body.appendChild( bttn );
+
+function fetchBooks() {
+    const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json';
+    fetch( booksJSON )
+        .then( response => response.json() )
+        .then( processBooks )
+        .then( removeBtn )
+        .then( addArrows )
+}
+
+// processBooks function
+var list = document.createElement( 'ul' );
+
+var processBooks = booksJSON => {
+   
+   booksJSON.forEach( function( el, index ) {
+       var lists = document.createElement( 'li' );
+       lists.setAttribute( 'id', booksJSON[index].id );
```

Comment:
Wouldn't it be easier to just access the current book from the loop with the `el` variable. Also I would rename `el` to `book` for clarity